user_guide_src/source/libraries/validation.rst

```diff
@@ -686,6 +686,8 @@ valid_url_strict        Yes        Fails if field does not contain a valid URL.
 valid_date              No         Fails if field does not contain a valid date. valid_date[d/m/Y]
                                    Accepts an optional parameter to matches
                                    a date format.
+valid_jwt               No         Fails if field contains anything other than
+                                   JWT Format.
```

Comment:
I know it seems silly, but I think we should add a note like: "Note: this does not verify the token itself, just the string format". Because people don't look at things sometimes. Or think.